build.yaml

```diff
@@ -2455,3 +2455,7 @@ node_modules:
   - src/node/ext/server.cc
   - src/node/ext/server_credentials.cc
   - src/node/ext/timeval.cc
+python_dependencies:
```

Comment:
This should probably be called `python_package` or something, to match the Node and Ruby entries. That will make it clearer if we ever need to list something besides dependencies here.